### ISSUE TITLE
the default value of ZoomScaleSheetLayoutView is not always 100

### ISSUE THREAD

**Issue by koowashi** [ClosedXML/ClosedXML#1579]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature
- [ ] Question

**Did you test against the latest CI build?**
- [ ] Yes
- [x] No

If you answered `No`, please test with [the latest development build](https://ci.appveyor.com/project/ClosedXML/ClosedXML/branch/develop/artifacts) first.

**Version of ClosedXML**

0.95.3

**What is the current behavior?**

if `ZoomScaleSheetLayoutView == 100` then it is overridden as `null`

**What is the expected behavior or new feature?**

Do not override the value.
At least my Excel's default zoom scale is 60%, so there is no way to set 100% due to that behavior.

Or possibly, the type of `ZoomScale` should be `Nullable<int>`, and `null` for the default value.

**Is this a regression from the previous version?**

No

## Reproducibility

I'm not sure, but it might be Japanese Excel specific issue.

- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Comment by igitur** [comment]
Please complete the full issue template with code to reproduce.

**Comment by koowashi** [comment]
```cs
using System;
using ClosedXML.Excel;

namespace ClosedXmlEg
{
    class Program
    {
        static void Main(string[] args)
        {
            var path = @"";
            using var workbook = new XLWorkbook(path);
            foreach (var worksheet in workbook.Worksheets)
            {
                worksheet.SheetView.ZoomScaleSheetLayoutView = 100;
            }
            workbook.Save();
        }
    }
}
```

https://github.com/ClosedXML/ClosedXML/blob/b8868c8f534c03086cd3dbe43617c459d2d0b584/ClosedXML/Excel/XLWorkbook_Save.cs#L4847-L4849

Isn't `100` and `null` different?

**Comment by igitur** [comment]
>   var path = @"";

Attach your file.

**Comment by koowashi** [comment]
It's nothing special. Just create a new one.

**Comment by Pankraty** [comment]
@koowashi it would be _much_ easier for everybody involved if you'd make it like this:

**Reproduction steps**
1. Set Excel default Zoom factor to 60% - **Screenshot attached**
2. Open a sample file in Excel - **Sample file attached**
3. See that `Worksheet1` has zoom factor 60% (by default) and `Worhsheet2` has zoom factor 100% (as specified in file)
4. Generate a similar file with ClosedXML - **Fully runnable code sample attached**
5. Open the produced file in Excel

**Expected result**
`Worksheet1` has zoom factor 60% (by default) and `Worhsheet2` has zoom factor 100% (as specified in file)

**Actual result**
Both worksheets have zoom factor of 60%
**Invalid file is also attached**

**Comment by koowashi** [comment]
I don't understand why you need the file.  Isn't it necessary to reproduce Excel itself, not the file? What you want to know is the behavior of Excel when the zoom scale is not specified?
I'm using Microsoft 365 subscription. 

I just want an API that bypasses L4847-4848.
In other words, an API that explicitly specifies 100%.

Please let me know if there is a good reason to replace the explicit 100% specification with undefined. I’ll fork.

**Comment by Pankraty** [comment]
I don't understand why it is that hard to follow the instructions in the issue template. You are asking for help fixing the library you use, we are asking for some assistance in making it slightly easier for the maintainers. And we've already spent on communications several times more than it would take to create a sufficient bug report from the beginning.

**Comment by koowashi** [comment]
Thanks to everyone involved in this great library.  I will follow the issue template when I have time.

For those who face the same problem, I'll leave one of the solutions.
You can create a file with 100% zoom scale by deleting these two lines: https://github.com/ClosedXML/ClosedXML/blob/b8868c8f534c03086cd3dbe43617c459d2d0b584/ClosedXML/Excel/XLWorkbook_Save.cs#L4847-L4848
